mathics/builtin/options.py

```diff
@@ -113,12 +130,13 @@ def matched():
         return ListExpression(*list(matched()))
 
 
+# Has this been removed from WL? I cannot find a WMA link.
 class NotOptionQ(Test):
     """
     <dl>
-    <dt>'NotOptionQ[$expr$]'
-        <dd>returns 'True' if $expr$ does not have the form of a valid
-        option specification.
+      <dt>'NotOptionQ[$expr$]'
+      <dd>returns 'True' if $expr$ does not have the form of a valid \
```

Comment:
Note that `\` is the way to add a soft line break.

Reply:
this works if the docstring is not a raw docstring, isn't it?

Reply:
Yes, you are correct, not raw When I start out with `r"""`  the backslash appears in the Django doc output .